### PR TITLE
Add Rarity

### PR DESCRIPTION
**Pull request by gbrad777**
ignore game part of branch, no games from me here

### CODE REVIEW

File: TcgPocket/Features/Games/Game.cs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 using TcgPocket.Features.Attributes;
 using TcgPocket.Features.Sets;
 using TcgPocket.Shared.Interfaces;
+using System.Net;
+using TcgPocket.Features.Rarities;
```
Comment (xalluna) [inline TcgPocket/Features/Games/Game.cs:6]: Get rid of the unused usings and add the following so it's not trying to use `System.Attribute`
```
using Attribute = TcgPocket.Features.Attributes.Attribute;
```

File: TcgPocket/Features/Games/Game.cs [hunk 2]
```diff
@@ -10,6 +12,7 @@ public class Game : GameGetDto, IEntity
     public List<CardType> CardTypes { get; set; }
     public List<Attributes.Attribute> Attributes { get; set; }
```
Comment (xalluna) [inline TcgPocket/Features/Games/Game.cs:13]: Do this in conjunction with the above comment. I didn't catch this on Zoe's PR.. mb

```suggestion
    public List<Attribute> Attributes { get; set; }
```

**Review by xalluna — COMMENTED** [review]
This is in almost every command/query so I'm just making a general comment. All of the dependecies that get injected should be `private readonly` not `public readonly`

**Review by xalluna — COMMENTED** [review]
General *NIT*: wrapping code in namespaces increases indentation and makes things a little more unpleasent to read compared to top level namespaces

**Comment by gbrad777** [comment]
> This is in almost every command/query so I'm just making a general comment. All of the dependecies that get injected should be `private readonly` not `public readonly`

yeah that was just me bein silly, i went on auto pilot and then copy pasted everywhere

**Comment by gbrad777** [comment]
> General _NIT_: wrapping code in namespaces increases indentation and makes things a little more unpleasent to read compared to top level namespaces

very nitpicky, respectfully will not be changing lol

**Review by xalluna — APPROVED** [review]

**Pull request merged**